ui/lib/pki/addon/routes/index.js

```diff
@@ -0,0 +1,10 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PkiRoute extends Route {
+  @service router;
+
+  redirect() {
+    this.router.transitionTo('vault.cluster.secrets.backend.pki.overview');
+  }
+}
```

Comment:
I navigated to the root initially and noticed the blank page so I added a redirect to the overview route.